.github/workflows/daily-testgrid.yaml

```diff
@@ -0,0 +1,43 @@
+name: testGrid-daily
+on:
+  schedule:
+  - cron: "0 0 * * 1-5"
+
+defaults:
+  run:
+    working-directory: ./testgrid/tgrun
+
+jobs:
+  testgrid-init:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: kubeconfig
```

Comment:
Why is this needed?  How will this work when there are > testgrid servers/

Reply:
Good points,
I was intending to add `vmi` cleanup to the *main loop* of `tgrun` left it here until then, but yes at this point it is confusing and multiple tg clusters would be hard to support with this model.

removing this in the next commit and will get clean up managed by `tgrun`.